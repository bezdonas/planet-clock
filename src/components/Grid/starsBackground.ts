/* tslint:disable */
export default 'data:image/gif;base64,R0lGODlhkAGQAYcAAAAAABoaGiYmJg0NDQICAkBAQAUFBQcHBxcXFw8PDwMDAwoKCgEBAXBwcD8/PwgICNDQ0Ht7eyIiIjMzMzY2NgYGBh8fHzw8PBQUFFZWVqCgoDAwMAwMDAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH/C05FVFNDQVBFMi4wAwEAAAAh+QQEAAAAACwAAAAAkAGQAQAI/wABCBxIsKDBgwgTKlzIsKHDhxAjSpxIsaLFixgzatzIsaPHjyBDihxJsqTJkyhTqlzJsqXLlzBjypxJs6bNmzhz6tzJs6fPn0CDCh1KtKjRo0iTKl3KtKnTp1CjSp1KtarVq1izat3KtavXr2DDih1LtqzZs2jTTkwgIIHat3CZBigQIK7du0MNDDCAt6/fv4ADCx5MuLDhw4gTK17MuLHjx5AjS55MubLly5gza97MubPnz6BDix5NurTp06hFKhjwILVrqQMKCHhN26mBAG5r697Nu7fv38CDCx9OvLjx48iTK1/OvLnz59CjS59Ovbp1qgsC8L3OveHcAd3DK//Mvl28+fPo06tfz769+/fw48ufT7++/Y4MEFAAf9+5ggYQONCfcwRc0IAEAxJYAQMJNujggxBGKOGEFFZo4YUYZqjhhhx26OGHIIYo4ogklmjiiSimqOKKLLbo4oswxijjjDTWaGNtCMx2o2UCFLDjj0AGKeSQRBZp5JFIJqnkkkw26eSTUAJnwQRRDjZllVhmqeWWXHbp5ZdghinmmGSWaeaZOumlAJpkfcfmWGy19uacdNZp55145qnnnnz26eefgAYq6KCEFmrooYgmquiijDbq6KOQRirppJRWaumlmGaq6aacdurpp6CGKiqJBFRAwKgwESDBgaei2pICDgT/uKarLOVHAQIM0qrrrrz26uuvwAYr7LDEFmvsscgmq+yyzDbr7LPQRistiORNS5Gb1kpUbbbcduvtt+CGK+645JZr7rnopqvuuuy26+678CKFAQbuTkBluwssEO++/Pbr778AByzwwAQXbPDBCCes8MIMN+zwwxBHLPHEFFds8cUYZ6zxxhx37PHHIIcs8sgkl2zyySinrPLKLLfs8sswxyzzzIPl6G6PNOes88489+zzz0AHLbSjNreL89BIJ6300kw37fTTUEctdXQP8Mfu0ewmUNfUXHft9ddghy322GSXbfbZaKet9tqXEdDqugY0kMGs6nIAgQb6rkuAAxu8a83234AHLvjghBdu+OFzHnBAuwdEEMHi6zb+eE1ad6q4TVivWzXinB9cudE+trt556SXbvrpqKeu+uqst+7667DHLvvstNdu++2456777rz3zpWtVqf7X4DsFnhgu6Xm6vvyzDfv/PPQpxcQADs=';
