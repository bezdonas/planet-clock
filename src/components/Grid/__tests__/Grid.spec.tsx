import { shallow } from 'enzyme';
import * as React from 'react';
import Grid from '../Grid';

it('Check snapshot', () => {
  const wrapper = shallow(<Grid>Hello world!</Grid>);
  expect(wrapper).toMatchSnapshot();
});
