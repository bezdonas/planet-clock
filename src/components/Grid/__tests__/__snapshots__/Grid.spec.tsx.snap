// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Grid>
    Hello world!
</Grid>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": "Hello world!",
    },
    "ref": null,
    "rendered": "Hello world!",
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  font-family: sans-serif;
  background: url(",
          "data:image/gif;base64,R0lGODlhkAGQAYcAAAAAABoaGiYmJg0NDQICAkBAQAUFBQcHBxcXFw8PDwMDAwoKCgEBAXBwcD8/PwgICNDQ0Ht7eyIiIjMzMzY2NgYGBh8fHzw8PBQUFFZWVqCgoDAwMAwMDAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH/C05FVFNDQVBFMi4wAwEAAAAh+QQEAAAAACwAAAAAkAGQAQAI/wABCBxIsKDBgwgTKlzIsKHDhxAjSpxIsaLFixgzatzIsaPHjyBDihxJsqTJkyhTqlzJsqXLlzBjypxJs6bNmzhz6tzJs6fPn0CDCh1KtKjRo0iTKl3KtKnTp1CjSp1KtarVq1izat3KtavXr2DDih1LtqzZs2jTTkwgIIHat3CZBigQIK7du0MNDDCAt6/fv4ADCx5MuLDhw4gTK17MuLHjx5AjS55MubLly5gza97MubPnz6BDix5NurTp06hFKhjwILVrqQMKCHhN26mBAG5r697Nu7fv38CDCx9OvLjx48iTK1/OvLnz59CjS59Ovbp1qgsC8L3OveHcAd3DK//Mvl28+fPo06tfz769+/fw48ufT7++/Y4MEFAAf9+5ggYQONCfcwRc0IAEAxJYAQMJNujggxBGKOGEFFZo4YUYZqjhhhx26OGHIIYo4ogklmjiiSimqOKKLLbo4oswxijjjDTWaGNtCMx2o2UCFLDjj0AGKeSQRBZp5JFIJqnkkkw26eSTUAJnwQRRDjZllVhmqeWWXHbp5ZdghinmmGSWaeaZOumlAJpkfcfmWGy19uacdNZp55145qnnnnz26eefgAYq6KCEFmrooYgmquiijDbq6KOQRirppJRWaumlmGaq6aacdurpp6CGKiqJBFRAwKgwESDBgaei2pICDgT/uKarLOVHAQIM0qrrrrz26uuvwAYr7LDEFmvsscgmq+yyzDbr7LPQRistiORNS5Gb1kpUbbbcduvtt+CGK+645JZr7rnopqvuuuy26+678CKFAQbuTkBluwssEO++/Pbr778AByzwwAQXbPDBCCes8MIMN+zwwxBHLPHEFFds8cUYZ6zxxhx37PHHIIcs8sgkl2zyySinrPLKLLfs8sswxyzzzIPl6G6PNOes88489+zzz0AHLbSjNreL89BIJ6300kw37fTTUEctdXQP8Mfu0ewmUNfUXHft9ddghy322GSXbfbZaKet9tqXEdDqugY0kMGs6nIAgQb6rkuAAxu8a83234AHLvjghBdu+OFzHnBAuwdEEMHi6zb+eE1ad6q4TVivWzXinB9cudE+trt556SXbvrpqKeu+uqst+7667DHLvvstNdu++2456777rz3zpWtVqf7X4DsFnhgu6Xm6vvyzDfv/PPQpxcQADs=",
          ") repeat 50% 50% #000;
  box-sizing: border-box;
  min-height: 100vh;
  text-align: center;
  display: grid;
  grid-gap: 20px;
  grid-template-columns: repeat(4, 1fr);
  grid-template-rows: repeat(2, 1fr);
  @media (max-width: 850px) {
    grid-template-columns: repeat(2, 1fr);
    grid-template-rows: repeat(4, 1fr);
  }
  @media (max-width: 450px) {
    grid-template-columns: repeat(1, 1fr);
    grid-template-rows: repeat(8, 1fr);
  }
",
        ],
      },
      "displayName": "styled.div",
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": "Hello world!",
      },
      "ref": null,
      "rendered": "Hello world!",
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": undefined,
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
  font-family: sans-serif;
  background: url(",
            "data:image/gif;base64,R0lGODlhkAGQAYcAAAAAABoaGiYmJg0NDQICAkBAQAUFBQcHBxcXFw8PDwMDAwoKCgEBAXBwcD8/PwgICNDQ0Ht7eyIiIjMzMzY2NgYGBh8fHzw8PBQUFFZWVqCgoDAwMAwMDAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH/C05FVFNDQVBFMi4wAwEAAAAh+QQEAAAAACwAAAAAkAGQAQAI/wABCBxIsKDBgwgTKlzIsKHDhxAjSpxIsaLFixgzatzIsaPHjyBDihxJsqTJkyhTqlzJsqXLlzBjypxJs6bNmzhz6tzJs6fPn0CDCh1KtKjRo0iTKl3KtKnTp1CjSp1KtarVq1izat3KtavXr2DDih1LtqzZs2jTTkwgIIHat3CZBigQIK7du0MNDDCAt6/fv4ADCx5MuLDhw4gTK17MuLHjx5AjS55MubLly5gza97MubPnz6BDix5NurTp06hFKhjwILVrqQMKCHhN26mBAG5r697Nu7fv38CDCx9OvLjx48iTK1/OvLnz59CjS59Ovbp1qgsC8L3OveHcAd3DK//Mvl28+fPo06tfz769+/fw48ufT7++/Y4MEFAAf9+5ggYQONCfcwRc0IAEAxJYAQMJNujggxBGKOGEFFZo4YUYZqjhhhx26OGHIIYo4ogklmjiiSimqOKKLLbo4oswxijjjDTWaGNtCMx2o2UCFLDjj0AGKeSQRBZp5JFIJqnkkkw26eSTUAJnwQRRDjZllVhmqeWWXHbp5ZdghinmmGSWaeaZOumlAJpkfcfmWGy19uacdNZp55145qnnnnz26eefgAYq6KCEFmrooYgmquiijDbq6KOQRirppJRWaumlmGaq6aacdurpp6CGKiqJBFRAwKgwESDBgaei2pICDgT/uKarLOVHAQIM0qrrrrz26uuvwAYr7LDEFmvsscgmq+yyzDbr7LPQRistiORNS5Gb1kpUbbbcduvtt+CGK+645JZr7rnopqvuuuy26+678CKFAQbuTkBluwssEO++/Pbr778AByzwwAQXbPDBCCes8MIMN+zwwxBHLPHEFFds8cUYZ6zxxhx37PHHIIcs8sgkl2zyySinrPLKLLfs8sswxyzzzIPl6G6PNOes88489+zzz0AHLbSjNreL89BIJ6300kw37fTTUEctdXQP8Mfu0ewmUNfUXHft9ddghy322GSXbfbZaKet9tqXEdDqugY0kMGs6nIAgQb6rkuAAxu8a83234AHLvjghBdu+OFzHnBAuwdEEMHi6zb+eE1ad6q4TVivWzXinB9cudE+trt556SXbvrpqKeu+uqst+7667DHLvvstNdu++2456777rz3zpWtVqf7X4DsFnhgu6Xm6vvyzDfv/PPQpxcQADs=",
            ") repeat 50% 50% #000;
  box-sizing: border-box;
  min-height: 100vh;
  text-align: center;
  display: grid;
  grid-gap: 20px;
  grid-template-columns: repeat(4, 1fr);
  grid-template-rows: repeat(2, 1fr);
  @media (max-width: 850px) {
    grid-template-columns: repeat(2, 1fr);
    grid-template-rows: repeat(4, 1fr);
  }
  @media (max-width: 450px) {
    grid-template-columns: repeat(1, 1fr);
    grid-template-rows: repeat(8, 1fr);
  }
",
          ],
        },
        "displayName": "styled.div",
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
